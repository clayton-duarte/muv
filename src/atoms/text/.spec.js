import React from 'react';

import Component from './';

it('should render a component', () => {
  const wrapper = shallow(
    <Component />,
  ).dive();
  expect(wrapper.props().color).toEqual('#0082c8');
  expect(wrapper).toMatchSnapshot();
});

it('should render a component with a prop', () => {
  const wrapper = shallow(
    <Component strong />,
  ).dive();
  expect(wrapper).toMatchSnapshot();
});

it('should render a component with a color', () => {
  const wrapper = shallow(
    <Component color="black" />,
  ).dive();
  expect(wrapper).toMatchSnapshot();
});
