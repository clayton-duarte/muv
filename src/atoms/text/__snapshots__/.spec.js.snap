// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a component 1`] = `
<p
  className="sc-bdVaJa hQcjfZ"
  color="#0082c8"
  transform="normal"
>
  default
</p>
`;

exports[`should render a component with a color 1`] = `
<p
  className="sc-bdVaJa vyQnj"
  color="black"
  transform="normal"
>
  default
</p>
`;

exports[`should render a component with a prop 1`] = `
<p
  className="sc-bdVaJa evIKLC"
  color="#0082c8"
  transform="normal"
>
  default
</p>
`;
